### PR TITLE
Added Button Modifiers Sheet

### PR DESCRIPTION
**Pull request by tbraun1551**
Added ability to view sheet in button section that displays all the modifiers on a button and their impact on the buttons when they are changed.

### CODE REVIEW

**Comment by Nef10** [comment]
Hey @tbraun1551, thank you for your contribution!
One questions: you build with Xcode beta 4, right? I haven't had the chance to update yet and I am unable to open your sheet, it crashes the app. I'll give it another try tomorrow after updating.

However, from a fist look at your code I noticed a few things:
* You added your development team to the project file, please remove
* You added a bridging header - I don't see why this is needed, can it be removed?
* The current files use spaces and your newly added code uses tabs, please adjust this

Feel free to wait with the adjustments unit I can have a closer look.

Thank you again!

**Comment by tbraun1551** [comment]
Hey @Nef10, 

- I did use Xcode beta 4 as it was the latest. I am guessing that is the source of the crash as it build and runs fine on my Mac and iPhone both on beta. 
- To remove the development team I just change it to none in signing capabilities and reupload correct?
-Briding header was a mistake on my end that I forgot to remove. Should be fixed with latest upload. 
-While tabs are superior to spaces I will fix the problem. :)

Let me know if there is anything else I can change/fix. 
Also I have had problems with the gesture section on my device so working on fixing those now.

**Comment by tbraun1551** [comment]
Made changes requested by @Nef10 as well as removed Button Modifier Sheet on MacOS temporarily as it is causing the app to hang and crash.

**Comment by tbraun1551** [comment]
Something in the Button Modifiers is causing the app to crash on MacOS. Do you have any idea what may be causing it?
Ive done limited testing but cant seem to figure out what is causing it.

**Comment by Nef10** [comment]
SwiftUI often has problems having two sheets in one view. Maybe check if it is solved when you remove the other sheet?

**Comment by tbraun1551** [comment]
It causes the app to hang even when I have it show up as a view on its own (not apart of a sheet)

**Comment by tbraun1551** [comment]
Any update on the status of this Branch. I have other additions to the app such as Widgets that I wanted to submit in different branches.

**Comment by Nef10** [comment]
Thanks for the update. For me the app on macOS is now crashing every time I try to open the Indicators view

**Comment by tbraun1551** [comment]
Uploaded a new build. Hoping this fixes the crashing on Mac as I have not had any problems on my end.

File: Shared/Groupings/GesturesGroup.swift [hunk 1]
```diff
@@ -8,17 +8,19 @@
...
 
 struct GesturesGroup: View {
+    
+    @State private var tapGestureBlock = TapGestureBlock()
     var body: some View {
         Group{
-            SectionView(description: "A Gesture that counts the number of taps."){
-                TapGestureBlock()
+            SectionView(description: "A Gesture that counts the number of taps.") {
+                tapGestureBlock
```
Comment (aaryankotharii) [inline Shared/Groupings/GesturesGroup.swift:16]: Why is a State variable added for tapGesture ?

Comment (tbraun1551) [reply Shared/Groupings/GesturesGroup.swift:15]: Sorry, I was having problems with the gestures so I was working on changing that but ended up scrapping my additions. The var was just leftover. 
I re-uploaded without the State var.

File: Shared/Groupings/GridGroup.swift [hunk 2]
```diff
@@ -0,0 +1,20 @@
+//
```
[the remark this answers is not in the document]
Comment (tbraun1551) [reply Shared/Groupings/GridGroup.swift:1]: Was going to add a section showing new SwiftUI grids but scrapped the idea and forgot to remove the file. Should be fixed now.

File: Shared/Groupings/IndicatorsGroup.swift [hunk 3]
```diff
@@ -18,7 +21,20 @@ struct IndicatorsGroup: View {
...
+                        VStack {
+                            ProgressView("Downloading…", value: progressAmount, total: 100)
+                        }
+                        .onReceive(timer) { _ in
+                            if progressAmount != 100 {
+                                progressAmount += 2
+                            } else {
+                                print(progressAmount)
+                                do {
+                                    sleep(2)
```
[the remark this answers is not in the document]
Comment (tbraun1551) [reply Shared/Groupings/IndicatorsGroup.swift]: Fixed

**Comment by tbraun1551** [comment]
tvOS should properly compile now and I made the rest of your suggested changes.

**Comment by Nef10** [comment]
@tbraun1551 thanks again, I merged the PR 👍

**Pull request merged**